rasa/nlu/model.py

```diff
@@ -27,8 +27,6 @@
 from rasa.nlu.training_data import Message, TrainingData
 from rasa.nlu.utils import write_json_to_file
 
-MODEL_NAME_PREFIX = "nlu_"
```

Comment:
I guess, we can leave this constant here as it is only used in this file.

Reply:
Not sure whether there was any dangling merge conflicts, but I can't see this used anywhere in the file.

Reply:
it is used in line 235, you changed it to `NLU_MODEL_NAME_PREFIX`, but I guess we can revert the change

Reply:
Can we leave it as `NLU_MODEL_NAME_PREFIX`? As far as I can see its only used in this file and I think thats what it was called originally (?).

Reply:
originally it was called `MODEL_NAME_PREFIX` but sure we can change it to `NLU_MODEL_NAME_PREFIX`

Reply:
Happy to leave it as `NLU_MODEL_NAME_PREFIX`